### PR TITLE
Docs reorg and debranding

### PR DESCRIPTION
**Pull request by elena-kolevska**

### CODE REVIEW

File: docs/docs/quickstart.md [hunk 1]
```diff
@@ -164,7 +166,7 @@ TS.CREATE sensor1 LABELS region east
...
+Another useful feature of Redis TimeSeries is compacting data by creating a rule for downsampling (`TS.CREATERULE`). For example, if you have collected more than one billion data points in a day, you could aggregate the data by every minute in order to downsample it, thereby reducing the dataset size to 24 * 60 = 1,440 data points. You can choose one of the many available aggregation types in order to aggregate multiple data points from a certain minute into a single one. The currently supported aggregation types are: `avg, sum, min, max, range, count, first, last, std.p, std.s, var.p, var.s and twa`.
```
Comment (LiorKogan) [inline docs/docs/quickstart.md:169]: suggest removing "of Redis TimeSeries". 
Also please change "Downsampling" to "Compaction" and "Compaction rules" everywere. We don't need the dual terminology.

File: docs/docs/quickstart.md [hunk 2]
```diff
@@ -185,7 +187,7 @@ With this creation rule, datapoints added to the `sensor1` timeseries will be gr
 
 
 ## Filtering
-RedisTimeSeries allows to filter by value, timestamp and by labels:
+Redis Time Series allows to filter by value, timestamp and by labels:
```
Comment (LiorKogan) [inline docs/docs/quickstart.md:190]: Again - suggest removing "Redis Time Series". Consider rewriting as "You can filter data by..."

File: docs/docs/quickstart.md [hunk 3]
```diff
@@ -3,22 +3,24 @@ title: "Quickstart"
...
 
-RedisTimeSeries is available on all Redis Cloud managed services, including a completely free managed database up to 30MB.
+Redis Time Series are available on all Redis Cloud managed services, including a completely free managed database up to 30MB.
 
 [Get started here](https://redislabs.com/try-free/)
 
 
 ### Docker
-To quickly try out RedisTimeSeries, launch an instance using docker:
+To quickly try out Redis Time Series, launch an instance using docker:
```
Comment (LiorKogan) [inline docs/docs/quickstart.md:23]: ... launch an instance of Redis Stack using docker:

File: docs/docs/quickstart.md (hunk 2 above)
Comment (elena-kolevska) [reply docs/docs/quickstart.md:190]: Thanks. Changing to "You can filter yor time series by value, timestamp and labels"

File: docs/docs/quickstart.md [hunk 4]
```diff
@@ -3,22 +3,24 @@ title: "Quickstart"
...
 
 ## Setup
 
-You can either get RedisTimeSeries setup in the cloud, in a Docker container or on your own machine.
+You can get RedisTimeSeries setup in the cloud, in a Docker container or on your own machine.
 
 ### Redis Cloud
 
-RedisTimeSeries is available on all Redis Cloud managed services, including a completely free managed database up to 30MB.
+Redis Time Series are available on all Redis Cloud managed services, including a completely free managed database up to 30MB.
```
Comment (LiorKogan) [inline docs/docs/quickstart.md:17]: Time Series is ...

File: docs/docs/_index.md [hunk 5]
```diff
@@ -1,21 +1,24 @@
...
-RedisTimeSeries is a Redis module that adds a time series data structure to Redis.
+Redis Stack (specifically, its RedisTimeSeries module) adds a time series data structure to Redis.
 
 ## Features
 * High volume inserts, low latency reads
 * Query by start time and end-time
 * Aggregated queries (min, max, avg, sum, range, count, first, last, STD.P, STD.S, Var.P, Var.S, twa) for any time bucket
 * Configurable maximum retention period
-* Downsampling/compaction for automatically updated aggregated timeseries
+* Compaction/compaction for automatically updated aggregated timeseries
```
Comment (LiorKogan) [inline docs/docs/_index.md:21]: Compaction instead of compaction/compaction

**Review by LiorKogan — APPROVED** [review]

**Review by danni-m — APPROVED** [review]

**Pull request merged**